backend/app/sql/total_pop_udf.sql

```diff
@@ -0,0 +1,24 @@
+CREATE OR REPLACE FUNCTION get_total_population(document_id UUID)
+RETURNS TABLE (zone TEXT, total_pop BIGINT) AS $$
+DECLARE
+    gerrydb_table_name TEXT;
+    sql_query TEXT;
+BEGIN
+    SELECT gerrydb_table INTO gerrydb_table_name
```

Comment:
(o) I don't 100% understand why gerrydb tables are document-specific. Can we have a shared one that just joins directly on geoid? or is this a performance thing where each gerrydb table only has rows for the geoids in a given doc?

Reply:
Not document-specific! Multiple documents might use the same gerrydb table – this just tracks which table to use for the document. We'll want to use the "pick your geography" flow on the frontend to inform which gerrydb table we assign to the document.

I set it up this way so that the MGGG folks could easily support new metrics / endpoints w/o having to worry about potentially breaking changes / migrations to a single monolithic `GerryDBTable` with all the geoids. Thoughts?

Reply:
Yeah stronger contract makes sense – rn the CLI just loads any layer in a gpkg lol